### ISSUE TITLE
Creating new cloud resources using permissions granted for a previous resource request?

### ISSUE THREAD

**Issue by RobertKielty** [cncf/cluster#251]
We have a request from a maintainer who is already using the CIL for their project.  
Having had an account setup on Equinix they now have the permissions to create more resources there.

Should they formally request the creation of new resources on the CIL? 
Should we devise and document a new process to track additional resource requests where user account setup is not required?

**Comment by vielmetti** [comment]
@RobertKielty 

Yes, there should be a separate request for resources beyond the original ask.

I'd suggest starting a new issue using the template, the usual questions about how much infra is needed so that we can track and budget for it, and anything else relevant to the process.

It's better for all concerned if individual projects set up separate projects within the CNCF organization.
